File: tests/utils/python/random_funcs.py
from tests.utils.python.CustomClass import CustomClass

def multiply_by_ten(a: int) -> int:
    return a * 10

def add_two_ints(a: int, b: int) -> int:
    return a + b

def create_custom_class(value: int, my_name: str) -> CustomClass:
    return CustomClass(value, my_name)

def print_custom_class(custom_class: CustomClass):
    custom_class.print_name_and_value()

def add_three_ints(a: int, b: int, c: int) -> int:
    step1 = add_two_ints(a, b)
    step2 = add_two_ints(step1, c)
    return step2

File: tests/utils/ts/randomFuncs.ts
import CustomClassTS from "./CustomClassTS"

function multiplyByTen(a: number): number {
    return a * 10
}

function addTwoInts(a: number, b: number): number {
    return a + b
}

function createCustomClass(value: number, my_name: string): CustomClassTS {
    return new CustomClassTS(value, my_name)
}

function addThreeInts(a: number, b: number, c: number): number {
    const step1 = addTwoInts(a, b)
    const step2 = addTwoInts(step1, c)
    return step2
}

export { multiplyByTen, addTwoInts, createCustomClass, addThreeInts }